cli/utils.py

```diff
@@ -10,6 +10,11 @@
 import time
 import json
 
+own_dir = os.path.dirname(os.path.realpath(__file__))
+root = os.path.abspath(os.path.join(own_dir, os.pardir))
+sys.path.insert(0, root + "/bin")
+from roger import roger_version
```

Comment:
I don't think it's a good idea to import roger from bin. Can you move that method into this file instead?